test/test_notifier.py

```diff
@@ -168,6 +168,33 @@ def __init__(self, **kwargs):
             ],
             "res": [{}, {"test": "OK"}, {}, {}, {}, {}, {}, {}, {}],
         },
+        # -------------------------------------------------------
+        # Notification Tests
+        # -------------------------------------------------------
+
+        # Same test as before, but now with a notification added
+        {
+            "params": {
+                "name": "test",
+                "condition": "a",
+                "message": "OK",
+                "holdoff": [10.0, "seconds"],
+                "tag" : "Chirayu, Tg",
+                "token" : "/home/chirayurai/Code/apprise_demo/single-docker-container/mqtt-version/src/apprise.yaml"
+            },
```

Comment:
Well, I hope this hardcoded path is temporary here...

Reply:
Yes. I wanted proof of concept to work. Forgot to change the unit tests.